Modules/CLI/CreateDICOMSeries/CreateDICOMSeries.xml

```diff
@@ -111,16 +132,55 @@
       <label>Rescale intercept</label>
       <name>rescaleIntercept</name>
       <longflag>--rescaleIntercept</longflag>
-      <description><![CDATA[Rescale interscept [0028-1052]. Converts pixel values on disk to pixel values in memory. (Pixel value in memory) = (Pixel value on disk) * rescaleSlope + rescaleIntercept.  Default is 0.0. Data values are converted on write (the data is scaled and shifted so that the slope and interscept will bring it back to the current intensity range).]]></description>
+      <description><![CDATA[Rescale interscept (0028,1052). Converts pixel values on disk to pixel values in memory. (Pixel value in memory) = (Pixel value on disk) * rescaleSlope + rescaleIntercept.  Default is 0.0. Data values are converted on write (the data is scaled and shifted so that the slope and interscept will bring it back to the current intensity range).]]></description>
       <default>0.0000</default>
     </double>
     <double>
       <label>Rescale slope</label>
       <name>rescaleSlope</name>
       <longflag>--rescaleSlope</longflag>
-      <description><![CDATA[Rescale slope [0028-1053]. Converts pixel values on disk to pixel values in memory. (Pixel value in memory) = (Pixel value on disk) * rescaleSlope + rescaleInterscept.  Default is 1.0. Data values are converted on write (the data is scaled and shifted so that the slope and interscept will bring it back to the current intensity range).]]></description>
+      <description><![CDATA[Rescale slope (0028,1053). Converts pixel values on disk to pixel values in memory. (Pixel value in memory) = (Pixel value on disk) * rescaleSlope + rescaleInterscept.  Default is 1.0. Data values are converted on write (the data is scaled and shifted so that the slope and interscept will bring it back to the current intensity range).]]></description>
       <default>1.0000</default>
     </double>
+    <string>
+      <name>contentDate</name>
+      <longflag>--contentDate</longflag>
+      <description><![CDATA[The date of the image content (0008,0023)]]></description>
```

Comment:
Small nitpick - would be nice if "The date of the image content" and other similar occurrences were replaced with the actual attribute names from DICOM.

Reply:
DICOM tag name is already in the "label" field (that is displayed in Create DICOM Series module GUI). This CLI module is just a backend anyway, user interacts with the DICOM module's GUI.

Reply:
Yes, that is true. It is not critical.